charts/sourcegraph/templates/redis/redis-cache.Deployment.yaml

```diff
@@ -65,15 +65,7 @@ spec:
             command:
             - /bin/sh
             - -c
-            - |
-              #!/bin/bash
-              response=$(
-                redis-cli ping
-              )
-              if [ "$response" != "PONG" ]; then
-                echo "$response"
-                exit 1
-              fi
+            - readiness.sh
```

Comment:
does this depend on the cwd? should we use `/readiness.sh` instead? Not a probe expert, so not sure how the WD is set

Reply:
Updated! I think it uses the root but this is more explicit regardless